pkgs/applications/office/jameica/default.nix

```
@@ -53,13 +55,16 @@ stdenv.mkDerivation rec {
     install -Dm644 plugin.xml $out/share/java/
     install -Dm644 build/jameica-icon.png $out/share/pixmaps/jameica.png
     cp ${desktopItem}/share/applications/* $out/share/applications/
+  '';
 
+  fixupPhase = ''
```

```suggestion
  postFixup = ''
```
we never want to overwrite such complicated and important phases

---

am i assuming right, that PRs that change this for other packages are then welcome? as i wrote there are others like that.

---

Most packages should not overwrite fixupPhase or patchPhase. There are probably some exceptions and we should comment those. Feel free to do some PRs to fix some things up.